doc/denite.txt

```diff
@@ -96,8 +96,8 @@ EXAMPLES						*denite-examples*
 	\ ['rg', '--files'])
 
 	" Change mappings.
-	call denite#custom#map('_', "\<C-j>", 'move_to_next_line')
-	call denite#custom#map('_', "\<C-k>", 'move_to_prev_line')
+	call denite#custom#map('_', '<C-j>', 'move_to_next_line')
```

Comment:
My bad. It worked on Neovim 0.1.6, Ubuntu 16.04, Gnome-terminal. Maybe the below?

1. The priority of custom '_' mapping is lower than mode mapping so `<C-j>` is override by a default mode mapping,